README.md

```diff
@@ -0,0 +1,25 @@
+# Colark
+
+All the Unstack services for [Colark](https://www.colark.com) software platform.
```

Comment:
This line is a bit confusing, although I like that you have a link here.  What does it do with Unstack services? Provide links, info, resources about it? You were right about this needing to be a short one-liner.

You could add a longer “description” section next, maybe with the text you had in View before it became “with no throwaway code”.

Reply:
@alodahl added the description you suggested but called it "About" - description felt a little formal when I typed it? If you think "Description" is better we can go with that!